packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -1,4 +1,4 @@
-const Page = require('@justeat/f-wdio-utils/src/page.object');
+const Page = require('../../../../../services/f-wdio-utils/src/page.object');
```

Comment:
This will be addressed in a 2nd PR once this has merged so that I can publish the updated f-wdio-utils to npm.